Client/Frontend/Browser/TabCell.swift

```diff
@@ -0,0 +1,269 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+
+class TabCell: UICollectionViewCell, TabTrayCell {
```

Comment:
I think it would be worth moving the TabTrayCell protocol and it's extension here too, since it's still in the GridTabViewController.

Reply:
Update: TabTrayCell protocol is in GridTabViewController while it's cells are elsewhere. We can move the protocol in TabCell or in it's own file I believe.